### PR TITLE
Fixing build issue

### PR DESCRIPTION
**Pull request by maria-camenzuli**

### CODE REVIEW

File: src/main/java/rocks/bastion/junit/BastionRunner.java [hunk 1]
```diff
@@ -29,13 +24,6 @@
 
     public BastionRunner(Class<?> testClass) throws InitializationError {
         super(testClass);
-        BastionFactory.setDefaultBastionFactory(new DefaultBastionFactory() {
```
Comment (KPull) [inline src/main/java/rocks/bastion/junit/BastionRunner.java]: So for this, you need to move this entire chunk of code here to _before_ `super.runChild()` below.

File: src/main/java/rocks/bastion/junit/BastionRunner.java [hunk 2]
```diff
@@ -85,6 +73,7 @@ protected Description describeChild(FrameworkMethod method) {
     protected void runChild(FrameworkMethod method, RunNotifier notifier) {
         runningTestCase = describeChild(method);
         currentNotifier = notifier;
+        BastionFactory.setDefaultBastionFactory(new DefaultBastionFactory());
```
Comment (KPull) [inline src/main/java/rocks/bastion/junit/BastionRunner.java]: This statement then needs to move to after `super.runChild()` below which clears the Bastion factory after the test has run.

Comment (maria-camenzuli) [inline src/main/java/rocks/bastion/junit/BastionRunner.java]: So the statement needs to be run both before and after `runChild`?

Comment (KPull) [inline src/main/java/rocks/bastion/junit/BastionRunner.java]: You need to have:

```
BastionFactory.setDefaultBastionFactory(new DefaultBastionFactory() {
            @Override
            protected void prepareBastion(BastionBuilderImpl<?> bastion) {
                registerModelConverters(bastion);
                bastion.registerListener(BastionRunner.this);
            }
        });
super.runChild();
BastionFactory.setDefaultBastionFactory(new DefaultBastionFactory());
```

Comment (maria-camenzuli) [inline src/main/java/rocks/bastion/junit/BastionRunner.java]: Aaaa ok. Making the changes required.

**Pull request merged**